namerd/iface/control-http/src/main/scala/io/buoyant/namerd/iface/AddrHandler.scala

```diff
@@ -6,7 +6,7 @@ import com.twitter.io.Buf
 import com.twitter.util.Future
 import io.buoyant.admin.names.DelegateApiHandler
 import io.buoyant.admin.names.DelegateApiHandler.{Addr => JsonAddr}
-import io.buoyant.namerd.RichActivity
+import io.buoyant.namerd.{RichActivity, stripTransformerPrefix}
```

Comment:
tioli:

I like to do

```scala
package io.buoyant.namerd
package iface
```

for this sort of thing so that intra-project imports aren't necessary.